### PR TITLE
Red Dead Revolver [SLUS-20500_E169BAF8] additional patches and correcting gametitle

### PR DESCRIPTION
**Pull request by ZippyZolton**
Been playing through the whole game with this and having no issues, so decided to push it here.

### CODE REVIEW

File: patches/SLUS-20500_E169BAF8.pnach [hunk 1]
```diff
@@ -1,10 +1,15 @@
-gametitle=Red Dead Revolver SLUS-20500 Blur orPost-Processing removal (V1.03)
-
+gametitle=Red Dead Revolver [SLUS-20500] (U)
```
Comment (pgert) [inline patches/SLUS-20500_E169BAF8.pnach:1]: "Red Dead Revolver [SLUS-20500] (U)" -> "Red Dead Revolver (V1.03) [SLUS-20500] (U)".
Make an empty line here to separate "gametitle ..." from "[Blur or Post-Processing removal]".

File: patches/SLUS-20500_E169BAF8.pnach [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-//https://forums.pcsx2.net/Thread-Post-your-PCSX2-cheats-patches-here?pid=611544#pid611544
+//Source: https://forums.pcsx2.net/Thread-Post-your-PCSX2-cheats-patches-here?pid=611544#pid611544
 patch=1,EE,004c4b78,word,03e00008
-patch=1,EE,004c4b7c,word,00000000
\ No newline at end of file
+patch=1,EE,004c4b7c,word,00000000
+
+[Force 60FPS]
+description=The original game is capped at 30FPS
+author=Alexkander_
```
Comment (pgert) [inline patches/SLUS-20500_E169BAF8.pnach:13]: author=asasega
https://forums.pcsx2.net/Thread-60-fps-codes?pid=548772#pid548772

**Comment by pgert** [comment]
Suggestion: Test these hacks to see if they work.

[Lower HUD]
patch=1,EE,2073BF14,word,3F866666

[Remove Blackbars]
patch=1,EE,2073C604,word,00000000

File: patches/SLUS-20500_E169BAF8.pnach (hunk 1 above)
Comment (ZippyZolton) [reply patches/SLUS-20500_E169BAF8.pnach]: done

File: patches/SLUS-20500_E169BAF8.pnach (hunk 2 above)
Comment (ZippyZolton) [reply patches/SLUS-20500_E169BAF8.pnach]: Thanks for giving me the real source, felt like moddb wasn't the right place for a pnach

**Comment by ZippyZolton** [comment]
> Suggestion: Test these hacks to see if they work.
> 
> [Lower HUD] patch=1,EE,2073BF14,word,3F866666
> 
> [Remove Blackbars] patch=1,EE,2073C604,word,00000000
Who made these? I will test them today

**Comment by ZippyZolton** [comment]
These two patches work fine, so I am adding them.
image

**Review by Mrlinkwii — APPROVED** [review]
assumed tested

**Pull request merged**